### PR TITLE
feat: kubeflow profiles upgrade to 1.7

### PR DESCRIPTION
**Pull request by i-chvets**
Summary of changes:
- Updated metadata with reference to 1.7 images
- Updated manifests templates.

### CODE REVIEW

File: src/templates/auth_manifests.yaml.j2 [hunk 1]
```diff
@@ -1,15 +1,73 @@
-kind: ClusterRole
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    kustomize.component: profiles
+  name: profiles-controller-service-account
+  namespace: {{ model_name }}
```
Comment (ca-scribner) [inline src/templates/auth_manifests.yaml.j2:7]: I think this needs to be replaced with whatever `ServiceAccount` is associated with the workload we deploy for kubeflow profiles.  In this case where it is a sidecar charm, I think we'd want to use the juju-provided `ServiceAccount` (which has the same name as the deployed app)

Comment (i-chvets) [reply src/templates/auth_manifests.yaml.j2:7]: The we will just not apply this `ServiceAccount` and we should be good as long as we have trust.

File: src/templates/auth_manifests.yaml.j2 [hunk 2]
```diff
@@ -1,15 +1,73 @@
...
+metadata:
+  labels:
+    kustomize.component: profiles
+  name: profiles-cluster-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- kind: ServiceAccount
```
[the remark this answers is not in the document]
Comment (i-chvets) [reply src/templates/auth_manifests.yaml.j2:71]: All we had in 1.6 is just this
```
kind: ClusterRole
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: {{ app_name }}
rules:
  - apiGroups:
      - "*"
    resources:
      - "*"
    verbs:
      - "*"
  - nonResourceURLs:
      - "*"
    verbs:
      - "*"
```

**Comment by i-chvets** [comment]
Resolved conflict to have metadata to point to 1.7rc images. Image rescan rebuild will be done for 1.7 at later stage.

File: src/templates/auth_manifests.yaml.j2 (hunk 2 above)
[the remark this answers is not in the document]
Comment (i-chvets) [reply src/templates/auth_manifests.yaml.j2:71]: RBAC will be revisited.

File: src/templates/auth_manifests.yaml.j2 (hunk 1 above)
Comment (i-chvets) [reply src/templates/auth_manifests.yaml.j2]: Resolved.

File: README.md [hunk 3]
```diff
@@ -1,7 +1,3 @@
 Charmed Kubeflow Profiles
 =========================
-
-This charm is part of the Kubeflow bundle. For instructions on how to deploy it,
-see https://charmed-kubeflow.io/docs.
-
-Upstream documentation can be found at https://www.kubeflow.org/docs/components/multi-tenancy/
+Charmed Kubeflow Profiles is the charm for [Kubeflow Multi-Tenancy](https://www.kubeflow.org/docs/components/multi-tenancy/) component. In [Charmed Kubeflow](https://charmed-kubeflow.io/), it is responsible for multi-user isolation and identity access management.
```
Comment (ca-scribner) [inline README.md:3]: What do you think about linking [this reference doc](https://charmed-kubeflow.io/docs/profile) here?  That way we're deferring most explanation to our docs.  For example, something like:
```suggestion
Charmed Kubeflow Profiles is the charm that implements the Kubeflow profile controller, which manages [user Profiles](https://charmed-kubeflow.io/docs/profile).  These Profiles enable [Kubeflow Multi-Tenancy](https://www.kubeflow.org/docs/components/multi-tenancy/) to allow for multi-user isolation.
```

**Review by ca-scribner — APPROVED** [review]

**Pull request merged**